style/client.css

```diff
@@ -99,6 +99,17 @@ hr {
 	border-right: 0;
 }
 
+button.button.disabled,
+button.button.disabled:hover,
+button.button.disabled:active {
```

Comment:
Should just be `.button.disabled`, not `button.button.disabled`.

This should also be moved to be right under `.button`.